extra-devel/musl/autobuild/build

```diff
@@ -0,0 +1,18 @@
+abinfo "Building musl... "
+
+./configure --prefix=/usr/lib/musl \
+    --exec-prefix=/usr \
+    --enable-wrapper=all \
+    CFLAGS="-ffat-lto-objects"
+
+make
+
+abinfo "Installing musl... "
+
+DESTDIR="$PKGDIR" make install
+
+mv "$PKGDIR"/lib/ld-musl*.so* "$PKGDIR"/usr/lib/
```

Comment:
This would have been handled by the ABTYPE.

Reply:
Done.